pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [unexpected SHOW STATISTICS results]

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11788228?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11788228?buildTab=artifacts#/) on master @ [28552c8876e8d6764c2102effbb8d9842185123d](https://github.com/cockroachdb/cockroach/commits/28552c8876e8d6764c2102effbb8d9842185123d):


```
=== RUN   TestImportFixture
    test_log_scope.go:167: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2803784559
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:104: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:117
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:321
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:324
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:117
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:321
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2803784559
--- FAIL: TestImportFixture (66.19s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/test-eng @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-31569

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11815753?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/11815753?buildTab=artifacts#/) on master @ [6cbd07ee6fbfb92706e8cdc8c559960b1bc41663](https://github.com/cockroachdb/cockroach/commits/6cbd07ee6fbfb92706e8cdc8c559960b1bc41663):


```
=== RUN   TestImportFixture
    test_log_scope.go:167: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture6166624
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:105: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:118
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:340
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:343
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:118
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:340
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:540: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture6166624
--- FAIL: TestImportFixture (62.38s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12099369?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12099369?buildTab=artifacts#/) on master @ [76a8d1077d3d29bcf020a48972efd1f77aa892c2](https://github.com/cockroachdb/cockroach/commits/76a8d1077d3d29bcf020a48972efd1f77aa892c2):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture591407346
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:540: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture591407346
--- FAIL: TestImportFixture (69.39s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12107792?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12107792?buildTab=artifacts#/) on master @ [330500d0433bae42bc7c0d6842a427aab3b27f03](https://github.com/cockroachdb/cockroach/commits/330500d0433bae42bc7c0d6842a427aab3b27f03):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture3058782210
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:540: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture3058782210
--- FAIL: TestImportFixture (69.51s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12122776?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12122776?buildTab=artifacts#/) on master @ [7e1f909e9f2ddcef2e418d6c2df11f3139a2b85a](https://github.com/cockroachdb/cockroach/commits/7e1f909e9f2ddcef2e418d6c2df11f3139a2b85a):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2572074790
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2572074790
--- FAIL: TestImportFixture (69.21s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12122777?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12122777?buildTab=artifacts#/) on master @ [7e1f909e9f2ddcef2e418d6c2df11f3139a2b85a](https://github.com/cockroachdb/cockroach/commits/7e1f909e9f2ddcef2e418d6c2df11f3139a2b85a):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture3452468123
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:540: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture3452468123
--- FAIL: TestImportFixture (68.34s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12138388?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12138388?buildTab=artifacts#/) on master @ [5af56c0ecd701d452c19fb89908012294b5336aa](https://github.com/cockroachdb/cockroach/commits/5af56c0ecd701d452c19fb89908012294b5336aa):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2704338973
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2704338973
--- FAIL: TestImportFixture (71.38s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12163357?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12163357?buildTab=artifacts#/) on master @ [8637692f77d69cda6052fef8e063cdac53870d3c](https://github.com/cockroachdb/cockroach/commits/8637692f77d69cda6052fef8e063cdac53870d3c):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture1016858373
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture1016858373
--- FAIL: TestImportFixture (71.12s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112164 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-foundations T-testeng branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12178033?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12178033?buildTab=artifacts#/) on master @ [f2bfa2a140fa302e79c9d31b84b944a6ac5e98ce](https://github.com/cockroachdb/cockroach/commits/f2bfa2a140fa302e79c9d31b84b944a6ac5e98ce):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture1299379637
    test_log_scope.go:81: use -show-logs to present logs inline
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:540: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture1299379637
--- FAIL: TestImportFixture (62.16s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112164 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-foundations T-testeng branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12213161?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12213161?buildTab=artifacts#/) on master @ [45f6344a2fc7d2d52f410e1ecad98f8c928cfeea](https://github.com/cockroachdb/cockroach/commits/45f6344a2fc7d2d52f410e1ecad98f8c928cfeea):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture1609192347
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:540: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture1609192347
--- FAIL: TestImportFixture (66.81s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112164 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-foundations T-testeng branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12231442?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12231442?buildTab=artifacts#/) on master @ [a45a76f76ee5d7bf87f5e642410d9be497caa051](https://github.com/cockroachdb/cockroach/commits/a45a76f76ee5d7bf87f5e642410d9be497caa051):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture3533954133
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture3533954133
--- FAIL: TestImportFixture (68.87s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112164 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [unexpected SHOW STATISTICS results] [C-test-failure O-robot T-sql-queries T-testeng branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12249346?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12249346?buildTab=artifacts#/) on master @ [fbee0764a4a2953a1178ea8c3edd352545caa227](https://github.com/cockroachdb/cockroach/commits/fbee0764a4a2953a1178ea8c3edd352545caa227):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2855350485
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2855350485
--- FAIL: TestImportFixture (69.89s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112164 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [unexpected SHOW STATISTICS results] [C-test-failure O-robot T-sql-queries T-testeng branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12249347?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12249347?buildTab=artifacts#/) on master @ [fbee0764a4a2953a1178ea8c3edd352545caa227](https://github.com/cockroachdb/cockroach/commits/fbee0764a4a2953a1178ea8c3edd352545caa227):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2402897804
    test_log_scope.go:81: use -show-logs to present logs inline
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:540: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2402897804
--- FAIL: TestImportFixture (59.38s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112164 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [unexpected SHOW STATISTICS results] [C-test-failure O-robot T-sql-queries T-testeng branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12267060?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12267060?buildTab=artifacts#/) on master @ [40c1ff68bbbe3e5f123c9147c3d43796c2935f55](https://github.com/cockroachdb/cockroach/commits/40c1ff68bbbe3e5f123c9147c3d43796c2935f55):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture4118782863
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture4118782863
--- FAIL: TestImportFixture (69.83s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112749 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-foundations T-testeng branch-release-23.1.12-rc release-blocker]
- #112164 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [unexpected SHOW STATISTICS results] [C-test-failure O-robot T-sql-queries branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12267061?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12267061?buildTab=artifacts#/) on master @ [40c1ff68bbbe3e5f123c9147c3d43796c2935f55](https://github.com/cockroachdb/cockroach/commits/40c1ff68bbbe3e5f123c9147c3d43796c2935f55):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture4140625332
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:540: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture4140625332
--- FAIL: TestImportFixture (73.11s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112749 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-foundations T-testeng branch-release-23.1.12-rc release-blocker]
- #112164 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [unexpected SHOW STATISTICS results] [C-test-failure O-robot T-sql-queries branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12282883?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12282883?buildTab=artifacts#/) on master @ [e945e0dfa7aa644fec63d2550acbf137d1f3f77b](https://github.com/cockroachdb/cockroach/commits/e945e0dfa7aa644fec63d2550acbf137d1f3f77b):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture394066198
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture394066198
--- FAIL: TestImportFixture (73.54s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112749 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1.12-rc release-blocker]
- #112164 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [unexpected SHOW STATISTICS results] [C-test-failure O-robot T-sql-queries branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12322705?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12322705?buildTab=artifacts#/) on master @ [aabfe2e105d1633d110050c236c8e0d8c107e168](https://github.com/cockroachdb/cockroach/commits/aabfe2e105d1633d110050c236c8e0d8c107e168):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2080986705
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1576
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1598
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2080986705
--- FAIL: TestImportFixture (71.64s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112749 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1.12-rc]
- #112164 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [unexpected SHOW STATISTICS results] [C-test-failure O-robot T-sql-queries branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12341910?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12341910?buildTab=artifacts#/) on master @ [8315a4bc997fb8b8d679079e14d6d7ca94d53bc6](https://github.com/cockroachdb/cockroach/commits/8315a4bc997fb8b8d679079e14d6d7ca94d53bc6):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture3618669263
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1595
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:523: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture3618669263
--- FAIL: TestImportFixture (73.12s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112749 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1.12-rc]
- #112164 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [unexpected SHOW STATISTICS results] [C-test-failure O-robot T-sql-queries branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12341911?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12341911?buildTab=artifacts#/) on master @ [8315a4bc997fb8b8d679079e14d6d7ca94d53bc6](https://github.com/cockroachdb/cockroach/commits/8315a4bc997fb8b8d679079e14d6d7ca94d53bc6):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture1687295643
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1595
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:541: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture1687295643
--- FAIL: TestImportFixture (70.56s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112749 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1.12-rc]
- #112164 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [unexpected SHOW STATISTICS results] [C-test-failure O-robot T-sql-queries branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12361207?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12361207?buildTab=artifacts#/) on master @ [40bcea4edc9d267d449df361582a30a80756c928](https://github.com/cockroachdb/cockroach/commits/40bcea4edc9d267d449df361582a30a80756c928):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2295745807
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1595
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:523: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2295745807
--- FAIL: TestImportFixture (73.14s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #112749 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1.12-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12371697?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12371697?buildTab=artifacts#/) on master @ [ca8fdcce91049cf98a9002047313884b32efa4c6](https://github.com/cockroachdb/cockroach/commits/ca8fdcce91049cf98a9002047313884b32efa4c6):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2398915672
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1595
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:523: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2398915672
--- FAIL: TestImportFixture (73.60s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #113129 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.2 release-blocker]
- #112749 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1.12-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12377536?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12377536?buildTab=artifacts#/) on master @ [3e029ff99a06e8780d363d8164b997a97b14b26d](https://github.com/cockroachdb/cockroach/commits/3e029ff99a06e8780d363d8164b997a97b14b26d):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture706375827
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1595
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:523: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture706375827
--- FAIL: TestImportFixture (72.29s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #113129 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.2 release-blocker]
- #112749 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1.12-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12377537?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12377537?buildTab=artifacts#/) on master @ [3e029ff99a06e8780d363d8164b997a97b14b26d](https://github.com/cockroachdb/cockroach/commits/3e029ff99a06e8780d363d8164b997a97b14b26d):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture274984293
    test_log_scope.go:81: use -show-logs to present logs inline
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1595
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:541: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture274984293
--- FAIL: TestImportFixture (63.67s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #113129 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.2 release-blocker]
- #112749 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1.12-rc]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12393031?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12393031?buildTab=artifacts#/) on master @ [aa85c69776da942b4d539891725b780ba995401e](https://github.com/cockroachdb/cockroach/commits/aa85c69776da942b4d539891725b780ba995401e):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture285328185
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1595
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:523: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture285328185
--- FAIL: TestImportFixture (73.22s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #113210 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1]
- #113129 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.2 release-blocker]
- #112749 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1.12-rc.FROZEN]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12393032?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12393032?buildTab=artifacts#/) on master @ [aa85c69776da942b4d539891725b780ba995401e](https://github.com/cockroachdb/cockroach/commits/aa85c69776da942b4d539891725b780ba995401e):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2695771130
    test_log_scope.go:81: use -show-logs to present logs inline
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1595
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:541: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture2695771130
--- FAIL: TestImportFixture (58.26s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #113210 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1]
- #113129 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.2 release-blocker]
- #112749 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1.12-rc.FROZEN]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12403241?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12403241?buildTab=artifacts#/) on master @ [aa85c69776da942b4d539891725b780ba995401e](https://github.com/cockroachdb/cockroach/commits/aa85c69776da942b4d539891725b780ba995401e):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture882920482
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1595
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:523: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture882920482
--- FAIL: TestImportFixture (73.33s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #113210 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1]
- #113129 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.2 release-blocker]
- #112749 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1.12-rc.FROZEN]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12413647?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12413647?buildTab=artifacts#/) on master @ [aa85c69776da942b4d539891725b780ba995401e](https://github.com/cockroachdb/cockroach/commits/aa85c69776da942b4d539891725b780ba995401e):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture1382657739
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1595
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:523: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture1382657739
--- FAIL: TestImportFixture (75.44s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #113210 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1]
- #113129 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.2 release-blocker]
- #112749 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-queries branch-release-23.1.12-rc.FROZEN]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12430625?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12430625?buildTab=artifacts#/) on master @ [71aaf665b6348b9b6e40f38a669bb68f83ea97cd](https://github.com/cockroachdb/cockroach/commits/71aaf665b6348b9b6e40f38a669bb68f83ea97cd):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture338806061
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:216
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1595
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:523: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture338806061
--- FAIL: TestImportFixture (73.10s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://tanzanite.cluster.engflow.com/invocation/898b2b3d-e9a2-4a85-a7b2-8b7e70832907) on master @ [71aaf665b6348b9b6e40f38a669bb68f83ea97cd](https://github.com/cockroachdb/cockroach/commits/71aaf665b6348b9b6e40f38a669bb68f83ea97cd):

Fatal error:

```
panic: test timed out after 55s
running tests:
	TestImportFixture (55s)

```
Stack: 

```
goroutine 4881018 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2259 +0x3b9
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x2d
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: outputs.zip/logTestImportFixture1429231852
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:124: automatically injected virtual cluster under test; see comment at top of test_server_shim.go for details.
```
</p>
</details>
<p>Parameters: <code>attempt=1</code>
, <code>run=14</code>
, <code>shard=1</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #113388 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-foundations T-testeng branch-release-23.2 release-blocker]
- #113386 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [C-test-failure O-robot T-sql-foundations T-testeng branch-release-23.1.12-rc.FROZEN release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

pkg/ccl/workloadccl/workloadccl_test.TestImportFixture [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12513536?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/12513536?buildTab=artifacts#/) on master @ [90a1f850ccfc9a123c9dcaba317296151ae32a8c](https://github.com/cockroachdb/cockroach/commits/90a1f850ccfc9a123c9dcaba317296151ae32a8c):


```
=== RUN   TestImportFixture
    test_log_scope.go:170: test logs captured to: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture1960936684
    test_log_scope.go:81: use -show-logs to present logs inline
    test_server_shim.go:93: cluster virtualization disabled due to issue: #110708 (expected label: C-bug)
    sql_runner.go:120: 
        	Error Trace:	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            				github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            				pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:221
        	Error:      	Received unexpected error:
        	            	query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	(1) attached stack trace
        	            	  -- stack trace:
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:366
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError.func1
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:75
        	            	  | github.com/cockroachdb/cockroach/pkg/util/retry.ForDuration
        	            	  | 	github.com/cockroachdb/cockroach/pkg/util/retry/retry.go:213
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils.SucceedsWithinError
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/soon.go:81
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).succeedsWithin
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:133
        	            	  | github.com/cockroachdb/cockroach/pkg/testutils/sqlutils.(*SQLRunner).CheckQueryResultsRetry
        	            	  | 	github.com/cockroachdb/cockroach/pkg/testutils/sqlutils/sql_runner.go:363
        	            	  | pkg/ccl/workloadccl/workloadccl_test_test.TestImportFixture
        	            	  | 	pkg/ccl/workloadccl/workloadccl_test_test/pkg/ccl/workloadccl/fixture_test.go:221
        	            	  | testing.tRunner
        	            	  | 	GOROOT/src/testing/testing.go:1595
        	            	  | runtime.goexit
        	            	  | 	src/runtime/asm_amd64.s:1650
        	            	Wraps: (2) query 'SELECT statistics_name, column_names, row_count, distinct_count, null_count
        	            	  |            FROM [SHOW STATISTICS FOR TABLE ingest.fx] WHERE row_count = 10': expected:
        	            	  | __auto__, {key}, 10, 10, 0
        	            	  | __auto__, {value}, 10, 1, 0
        	            	  |
        	            	  | got:
        	            	Error types: (1) *withstack.withStack (2) *errutil.leafError
    panic.go:541: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/fd719ddfcba889d931fc8ef278f553b7/logTestImportFixture1960936684
--- FAIL: TestImportFixture (60.51s)
```
<p>Parameters: <code>TAGS=bazel,gss</code>
, <code>stress=true</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #113743 pkg/ccl/workloadccl/workloadccl_test: TestImportFixture failed [unexpected SHOW STATISTICS results] [C-test-failure O-robot T-sql-queries branch-release-23.2 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestImportFixture.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/ccl/workloadccl/BUILD.bazel
+++ b/pkg/ccl/workloadccl/BUILD.bazel
@@ -44,6 +44,7 @@ go_test(
         "//pkg/security/securityassets",
         "//pkg/security/securitytest",
         "//pkg/server",
+        "//pkg/settings/cluster",
         "//pkg/sql/stats",
         "//pkg/testutils",
         "//pkg/testutils/serverutils",

--- a/pkg/ccl/workloadccl/fixture_test.go
+++ b/pkg/ccl/workloadccl/fixture_test.go
@@ -21,6 +21,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/base"
 	_ "github.com/cockroachdb/cockroach/pkg/ccl"
 	"github.com/cockroachdb/cockroach/pkg/ccl/workloadccl"
+	"github.com/cockroachdb/cockroach/pkg/settings/cluster"
 	"github.com/cockroachdb/cockroach/pkg/sql/stats"
 	"github.com/cockroachdb/cockroach/pkg/testutils"
 	"github.com/cockroachdb/cockroach/pkg/testutils/serverutils"
@@ -42,6 +43,9 @@ type fixtureTestGen struct {
 	flags workload.Flags
 	val   string
 	empty string
+	// tableAutoStatsEnabled, if set, enables auto stats collection at the table
+	// level.
+	tableAutoStatsEnabled bool
 }
 
 func makeTestWorkload() workload.Flagser {
@@ -67,9 +71,13 @@ func init() {
 func (fixtureTestGen) Meta() workload.Meta     { return fixtureTestMeta }
 func (g fixtureTestGen) Flags() workload.Flags { return g.flags }
 func (g fixtureTestGen) Tables() []workload.Table {
+	schema := `(key INT PRIMARY KEY, value INT)`
+	if g.tableAutoStatsEnabled {
+		schema += ` WITH (sql_stats_automatic_collection_enabled = true)`
+	}
 	return []workload.Table{{
 		Name:   `fx`,
-		Schema: `(key INT PRIMARY KEY, value INT)`,
+		Schema: schema,
 		InitialRows: workload.Tuples(
 			fixtureTestGenRows,
 			func(rowIdx int) []interface{} {
@@ -182,26 +190,27 @@ func TestImportFixture(t *testing.T) {
 	stats.DefaultRefreshInterval = time.Millisecond
 	stats.DefaultAsOfTime = 10 * time.Millisecond
 
+	// Disable auto stats collection on all tables. This is needed because we
+	// run only one auto stats job at a time, and collecting auto stats on
+	// system tables can significantly delay the collection of stats on the
+	// fixture after the IMPORT is done.
+	st := cluster.MakeTestingClusterSettings()
+	stats.AutomaticStatisticsClusterMode.Override(ctx, &st.SV, false)
+	stats.AutomaticStatisticsOnSystemTables.Override(ctx, &st.SV, false)
 	s, db, _ := serverutils.StartServer(t, base.TestServerArgs{
-		// Occasionally, for some reason auto stats aren't collected within the
-		// retry window after the import is finished in multi-tenant setup, so
-		// for now we disable this config.
-		DefaultTestTenant: base.TestIsForStuffThatShouldWorkWithSecondaryTenantsButDoesntYet(110708),
+		Settings:          st,
+		DefaultTestTenant: base.TestIsForStuffThatShouldWorkWithSharedProcessModeButDoesntYet(base.TestTenantProbabilistic, 113916),
 	})
 	defer s.Stopper().Stop(ctx)
 	sqlDB := sqlutils.MakeSQLRunner(db)
 
-	sqlDB.Exec(t, `SET CLUSTER SETTING sql.stats.automatic_collection.enabled=true`)
-
 	gen := makeTestWorkload()
+	// Enable auto stats only on the table being imported.
+	gen.(*fixtureTestGen).tableAutoStatsEnabled = true
 	flag := fmt.Sprintf(`val=%d`, timeutil.Now().UnixNano())
 	if err := gen.Flags().Parse([]string{"--" + flag}); err != nil {
 		t.Fatalf(`%+v`, err)
 	}
-	// Wait for the `ensureAllTables` unconditional auto stats to run before
-	// starting the test, so we don't hit a timing window where 2 sets of auto
-	// stats are collected.
-	time.Sleep(2 * time.Second)
 
 	const filesPerNode = 1
 

